packages/relative-values/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 4000",
```

Comment:
Cool, I'm fine with it, we should pick separate ports for each package. Then maybe we could do a single `npx turbo dev` on the top level of the repo for all subpackages (I've been thinking about it recently and will look into it later).

Reply:
Yea, that sounds good

Reply:
In this case I just needed something different from the Website.